test/e2e/seeder/anvil-seeder.js

```diff
@@ -0,0 +1,149 @@
+const { DEFAULT_FIXTURE_ACCOUNT, ENTRYPOINT } = require('../constants');
```

Comment:
this file supports the same functionality as ganache seeder, but adapted to anvil (meaning using publicClient/walletClient/testClient for the different actions)